config/autoload/zend-expressive.global.php

```diff
@@ -3,9 +3,6 @@
 return [
     'debug' => false,
 
-    // Cache the configuration. Recommended for production.
-    'config_cache_enabled' => false,
```

Comment:
I'm torn on this. One of the key reasons for having development mode was to solve the problem of ensuring it's optimized by default, but can be development ready otherwise. Having to _add_ configuration to make it production ready makes for educational overhead.

One possibility would be to have the setting commented out:

```php
    // Uncomment the below to enable configuration caching.
    // You may then use development-enable/development-disable
    // to toggle caching.
    // 'config_cache_enabled' => true,
```

Thoughts?